src/exchange/swap/getExchangeRates.ts

```diff
@@ -16,16 +16,19 @@ import {
 const getExchangeRates: GetExchangeRates = async (
   exchange: Exchange,
   transaction: Transaction,
-  userId?: string // TODO remove when wyre doesn't require this for rates
+  userId?: string, // TODO remove when wyre doesn't require this for rates
+  currencyTo?: TokenCurrency | CryptoCurrency | undefined | null
```

Comment:
yikes

Reply:
Yeah this is not ideal but I tried to keep it compatible with the old swap :/. 
I guess the alternative would have been to copy/paste the function and change the args which is not super nice either.

Reply:
nah, this is good, it's just ugly 🤣